src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1793,12 +1793,13 @@ private function getDefinitionsFromArguments(array $arguments, \SplObjectStorage
     private function dumpValue(mixed $value, bool $interpolate = true): string
     {
         if (\is_array($value)) {
+            $isList = array_is_list($value);
```

Comment:
I would move it after the `if` that does not need it.

Reply:
Oops, that makes much sense. Fixed.